### DISCUSSION TITLE
htmlspecialchars ERROR

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5699] [Q&A]
Please help suddenly an FATAL error appears:

htmlspecialchars() expects parameter 1 to be string, array given (View: D:\develop\bashaier\vendor\statamic\cms\resources\views\auth\layout.blade.php)

htmlspecialchars() expects parameter 1 to be string, array given (View: D:\develop\bashaier\vendor\statamic\cms\resources\views\auth\login.blade.php)

htmlspecialchars() expects parameter 1 to be string, array given (View: D:\develop\bashaier\vendor\statamic\cms\resources\views\auth\dashboard.blade.php)

I did not do anything

**Comment by jasonvarga** [comment]
You'll need to provide more info than that. You must have done _something_. Does it happen to you on a completely fresh install. Can you show a screenshot?

**Reply by HazemMohamed252** [reply]
It is solved now. Thank you

**Reply by jasonvarga** [reply]
What was the problem?

**Reply by HazemMohamed252** [reply]
This happened when i include another lang folder to custom my own project
